lib/string-checker.js

```diff
@@ -83,17 +83,6 @@ StringChecker.prototype = {
      * @throws Error on invalid source code.
      */
     _parse: function(str) {
```

Comment:
hmmm, perhaps instead of a static call to `JsFile.parse`, we simple move the parse call into the constructor of JsFile, along with passing `this._esprima` and `this._esprima` as options to JsFile?